src/graphql/event/kerberos.rs

```diff
@@ -77,20 +77,28 @@ impl BlockListKerberos {
         self.inner.proto
     }
 
-    async fn last_time(&self) -> i64 {
-        self.inner.last_time
+    /// The last time the event was seen in string wthin the range
+    /// representable by a `i64`.
+    async fn last_time(&self) -> StringNumber<i64> {
```

Comment:
@marioCluml @sophie-cluml,

I think we need to check if 'last_time' actually means what you described based on the protocol. If understanding the protocol takes time, it’s fine to omit this comment for now. The same approach should be applied to other protocol-specific terms.